.github/actions/docker-build/action.yaml

```diff
@@ -67,13 +67,18 @@ runs:
       with:
         env-file: .ci_env
 
+    - name: Normalize TAGS
+      shell: bash
```

Comment:
Could you please set the shell on pipeline-global level? It was forgotten here but we usually have it in all our pipelines :)
Should be simply:
```
defaults:
  run:
    shell: bash
```

Reply:
I tried to set the shell on pipeline-global level, as you described, but now the GitHub Actions are failing with the following error: 
```
Raffy23/gh-automation/fix/normalize_docker_tags/.github/actions/docker-build/action.yaml (Line: 73, Col: 7): Required property is missing: shell
```

Reply:
Aha! Sorry, my suggestion only works in pipelines but not in actions definitions so there's nothing to change here

Reply:
Ok, I reverted the last commit to undo the change